x/masterchef/keeper/abci.go

```diff
@@ -758,6 +759,8 @@ func (k Keeper) ProcessTakerFee(ctx sdk.Context) {
 		if err != nil {
 			ctx.Logger().Error("Failed to swap taker fee", "error", err)
```

Comment:
Can we figure out a way to avoid logging error when its expected? For example, rn its for paxg and eth taker fees

Reply:
Validators are reporting this as an issues